orderable/managers.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class OrderableManager(models.Manager):
+
+    def before(self, orderable):
+        return self.filter(sort_order__lt=orderable.sort_order).last()
+
+    def after(self, orderable):
+        return self.filter(sort_order__gt=orderable.sort_order).first()
```

Comment:
Just realised that this doesn't take into account pre-filtering unwanted results out of the QS. See `Orderable.next()`.